docs/adguard-for-android/solving-problems/adguard-outbound-proxy.md

```diff
@@ -0,0 +1,110 @@
+Proxy port | *9050*
+   
+Or you can tap on **Add proxy** and enter these values manually, then set the Orbot proxy as a default one.
+   
+5. Enable AdGuard protection.
+
+> Please note, AdGuard will now forward all traffic through Orbot proxy, if you disable Orbot proxy, then Internet connection will be unavailable until you disable outbound proxy settings in AdGuard
+
+### How to use AdGuard with PIA (Private Internet Access)
+
+*here we presume that you are already a PIA VPN client and have it installed on your device*
+
+1. Open AdGuard and go to **Settings -> Network -> Proxy settings**.
+
+2. Tap on the **Add proxy server** button and enter the following data:
+    
+Field      |  Value
+-------    | --------- 
+Proxy type | *SOCKS5* 
+Proxy host | *proxy-nl.privateinternetaccess.com*
+Proxy port | *1080*
+	
+3. You also need to fill out the **Username/Password** fields. To do so, log in to the [Client Control Panel](https://www.privateinternetaccess.com/pages/client-sign-in) on the PIA website. Click the **Generate Password** button under the **Generate PPTP/L2TP/SOCKS Password** section. A Username starting with "x" and a random Password will be shown. Use them to fill out the **Username** and **Password** fields in AdGuard.
+
+4. Set this proxy as a default one by toggling the switch, then click *Save*.
+
+5. Enable AdGuard protection.
+
+### How to use AdGuard with TorGuard
+
+*here we presume that you are already a TorGuard client and have it installed on your device*
+
+1. Open AdGuard and go to **Settings -> Network -> Proxy settings**.
+
+2. Tap on the **Add proxy server** button and enter the following data:
+
+Field      |  Value
+-------    | --------- 
+Proxy type | *SOCKS5* 
+Proxy host | *proxy.torguard.org* or *proxy.torguard.io*
+Proxy port | *1080* or *1085* or *1090*
+	
+3. For *Username* and *Password* fields, enter your proxy username and proxy password you have chosen at TorGuard signup.
+
+4. Set this proxy as a default one by toggling the switch, then click *Save*.
+
+5. Enable AdGuard protection.
+
+### How to use AdGuard with NordVPN
+
+1. Open AdGuard and go to **Settings -> Network -> Proxy settings**.
+
+2. Tap on the **Add proxy server** button and enter the following data:
+
+Field      |  Value
+-------    | --------- 
+Proxy type | *SOCKS5* 
+Proxy host | *any server save for "Obfuscated" from [this list](https://nordvpn.com/servers/)*
+Proxy port | *1080*
+	
+3. For *Username* and *Password* fields, enter your NordVPN Username and Password.
+
+4. Set this proxy as a default one by toggling the switch, then click *Save*.
+
+5. Enable AdGuard protection.
+
+### How to use AdGuard with Shadowsocks
+
+*here we presume that you have already configured a Shadowsocks server and a client on your device*
+
+**NOTE: You should remove Shadowsocks application from filtering before setting up the process (AdGuard - Apps management - Shadowsocks - Disable AdGuard protection) to avoid infinite loops and drops**
```

Comment:
```suggestion
Note: You should remove the Shadowsocks application from filtering before setting up the process (AdGuard -> Apps management -> Shadowsocks -> Disable AdGuard protection) to avoid infinite loops and drops.**
```